packages/payment/src/models/capture.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  BeforeCreate,
+  Entity,
+  ManyToOne,
+  OnInit,
+  OptionalProps,
+  PrimaryKey,
+  Property,
+} from "@mikro-orm/core"
+
+import { generateEntityId } from "@medusajs/utils"
+import Payment from "./payment"
+
+type OptionalCaptureProps = "created_by" | "created_at" | "completed_at"
+
+@Entity({ tableName: "capture" })
+export default class Capture {
+  [OptionalProps]?: OptionalCaptureProps
+
+  @PrimaryKey({ columnType: "text" })
+  id: string
+
+  @Property({
+    columnType: "numeric",
+    serializer: Number,
+  })
+  amount: number
+
+  @ManyToOne(() => Payment, {
+    onDelete: "cascade",
+    index: "IDX_capture_payment_id",
+    fieldName: "payment_id",
+  })
+  payment: Payment
+
+  @Property({
+    onCreate: () => new Date(),
+    columnType: "timestamptz",
+    defaultRaw: "now()",
+  })
+  created_at: Date
```

Comment:
suggestion: sometime we put as optional (?) and sometimes not cc @olivermrbl should we put it always optional to match the optional props?

Reply:
Yes, let's do that.